Classes/WPEditorViewController.m

```diff
@@ -174,6 +174,55 @@ - (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withT
     [self.toolbarView setNeedsLayout];
 }
 
+#pragma mark - Keyboard shortcuts
+
+- (BOOL)canBecomeFirstResponder
+{
+    return YES;
+}
+
+- (NSArray<UIKeyCommand *> *)keyCommands
+{
+    return @[
+             [UIKeyCommand keyCommandWithInput:@"I"
+                                 modifierFlags:UIKeyModifierCommand|UIKeyModifierShift
+                                        action:@selector(didTouchMediaOptions)
+                          discoverabilityTitle:NSLocalizedString(@"Insert Image", @"Discoverability title for insert image keyboard shortcut.")],
```

Comment:
This shortcut should be renamed to be "Insert Media" and maybe change the shortcut to Cmd + M